src/v2/Components/LotTimer.tsx

```diff
@@ -60,6 +59,8 @@ export const LotTimerFragmentContainer = createFragmentContainer(LotTimer, {
   `,
 })
 
+// If `extendedBiddingEndAt` is passed in and non-null, the `time`
+// parameter refers to the time left in the extended period.
```

Comment:
**suggestion (non-blocking):** Maybe there is a better name for the param that makes this clear?

Reply:
this follows a convention we have with SaleArtwork and Sale where it's end_at on each of those, thus we differentiate here with extended_bidding_end_at, but open to suggestions.